tests/testthat/test-unit-latent_gamma.R

```diff
@@ -0,0 +1,76 @@
+test_that("posterior_predict_latent_gamma outputs positive integers with length equal to draws", { # nolint: line_length_linter.
+  fit_gamma <- readRDS(
+    system.file("extdata/fit_gamma.rds", package = "epidist")
+  )
+  prep <- brms::prepare_predictions(fit_gamma)
+  i <- 1
+  pred_i <- posterior_predict_latent_gamma(i = i, prep)
+  expect_equal(floor(pred_i), pred_i)
+  expect_equal(length(pred_i), prep$ndraws)
+  expect_gte(min(pred_i), 0)
+})
+
+test_that("posterior_predict_latent_gamma errors for i out of bounds", { # nolint: line_length_linter.
+  fit_gamma <- readRDS(
+    system.file("extdata/fit_gamma.rds", package = "epidist")
+  )
+  prep <- brms::prepare_predictions(fit_gamma)
+  i_out_of_bounds <- length(prep$data$Y) + 1
+  expect_error(posterior_predict_latent_gamma(i = i_out_of_bounds, prep))
+})
+
+test_that("posterior_predict_latent_gamma can generate predictions with no censoring", { # nolint: line_length_linter.
+  fit_gamma <- readRDS(
+    system.file("extdata/fit_gamma.rds", package = "epidist")
+  )
+  draws <- data.frame(obs_t = 1000, pwindow_upr = 0, swindow_upr = 0) |>
+    tidybayes::add_predicted_draws(fit_gamma, ndraws = 100)
+  expect_equal(draws$.draw, 1:100)
+  pred <- draws$.prediction
+  expect_gte(min(pred), 0)
+  expect_true(all(abs(pred - round(pred)) > .Machine$double.eps^0.5))
+})
+
+test_that("posterior_predict_latent_gamma predicts delays for which the data is in the 95% credible interval", { # nolint: line_length_linter.
+  fit_gamma <- readRDS(
```

Comment:
if this is all we are using it for we could just run a fit in the setup.R file in testthat to get rid of the large saved file

Reply:
Yes it is all. This would be annoying in that I run `setup.R` before running tests some of the time but apart from that it's a good solution. I'll add it to the issue on large files

Reply:
https://github.com/epinowcast/epidist/issues/277#issuecomment-2327120067